pandora-server-directory/src/account/accountDirectMessages.ts

```diff
@@ -99,22 +99,22 @@ export class AccountDirectMessages {
 		}
 		const time = GetNextMessageTime();
 		const accounts = GetDirectMessageId(this._account, target);
-		const message: IDirectoryDirectMessage = {
+		const message = {
```

Comment:
You should always explicitly define type when creating object in-place like this - otherwise TS won't catch if you specify unknown key (as the inferred type will still be assignable where you use it, so it isn't an error)